tests/CMakeLists.txt

```diff
@@ -271,7 +271,7 @@ if(AVIF_BUILD_APPS)
                                               ${CMAKE_CURRENT_SOURCE_DIR}/data
     )
 
-    if(AVIF_ENABLE_AVIFGAINMAPUTIL AND AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION)
+    if(AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION)
```

Comment:
IMPORTANT: It seems that this should still be guarded by `AVIF_ENABLE_GAIN_MAP`.

Please try building with `-DAVIF_ENABLE_GAIN_MAP=OFF -DAVIF_BUILD_APPS=ON`.

Reply:
I understand why this is correct now. `AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION` implies `AVIF_ENABLE_GAIN_MAP`, so it is not necessary to test `AVIF_ENABLE_GAIN_MAP` here.